src/redux/sagas/motions/moveFundsMotion.ts

```diff
@@ -112,65 +97,197 @@ function* moveFundsMotion({
+          parentDomainNativeId: rootDomain.nativeId,
+          parentDomainSkillId: rootDomain.nativeSkillId,
+          domainNativeId: toDomain.nativeId,
+          domainSkillId: toDomain.nativeSkillId,
+        });
+
+        const encodedAction = colonyClient.interface.encodeFunctionData(
+          contractMethod,
+          [
+            ...(isOldVersion
+              ? []
+              : [multiSigPermissionDomainId, multiSigChildSkillIndex]),
+            userPermissionDomainId,
+            fromChildSkillIndex,
+            toChildSkillIndex,
+            fromPot,
+            toPot,
+            amount,
+            tokenAddress,
+          ],
+        );
+
+        return {
+          context: ClientType.MultisigPermissionsClient,
+          methodName: 'createMotion',
+          identifier: colonyAddress,
+          params: [
+            userPermissionDomainId,
+            userChildSkillIndex,
+            [colonyAddress],
+            [encodedAction],
+          ],
+          group: {
+            key: batchKey,
+            id: metaId,
+            index: 0,
+          },
+          ready: false,
+        };
+      }
+
+      const votingReputationClient = yield colonyClient.getExtensionClient(
+        Extension.VotingReputation,
+      );
+
+      const motionChildSkillIndex = yield call(
+        getChildIndex,
+        colonyClient.networkClient,
+        colonyClient,
+        createdInDomain.nativeId,
+        fromDomain.nativeId,
+      );
+
+      const { skillId } = yield call(
+        [colonyClient, colonyClient.getDomain],
+        createdInDomain.nativeId,
+      );
+
+      const { key, value, branchMask, siblings } = yield call(
+        colonyClient.getReputation,
+        skillId,
         AddressZero,
-        encodedAction,
-        key,
-        value,
-        branchMask,
-        siblings,
-      ],
-      group: {
-        key: batchKey,
-        id: metaId,
-        index: 0,
-      },
-      ready: false,
-    });
+      );
+
+      const [fromPermissionDomainId] = yield call(
+        getPermissionProofs,
+        colonyClient.networkClient,
+        colonyClient,
+        fromDomain.nativeId,
+        [ColonyRole.Funding],
+        votingReputationClient.address,
+      );
+
+      const [, fromChildSkillIndex] = yield call(
+        getPermissionProofs,
+        colonyClient.networkClient,
+        colonyClient,
+        fromDomain.nativeId,
+        [ColonyRole.Funding],
+        votingReputationClient.address,
+      );
+
+      const [, toChildSkillIndex] = yield call(
+        getPermissionProofs,
+        colonyClient.networkClient,
+        colonyClient,
+        toDomain.nativeId,
+        [ColonyRole.Funding],
+        votingReputationClient.address,
+      );
+
+      const encodedAction = colonyClient.interface.encodeFunctionData(
+        contractMethod,
+        [
+          ...(isOldVersion
+            ? []
+            : [createdInDomain.nativeId, constants.MaxUint256]),
```

Comment:
This `constants.MaxUint256` still feels like a work around to me rather than the actual childSkillIndex that is expected. I suspect it might also be the cause of being unable to finalize a motion created in a sub domain.